fsharp-backend/src/LibBinarySerialization/SerializedTypes.fs

```diff
@@ -196,6 +197,9 @@ module Handler =
     // If there's no module
     // CLEANUP: convert these into repl and get rid of this case
     | UnknownHandler of string * string * ids
+    /// Corresponds to HttpMiddlewareV1
+    /// HttpBytesTODO: rebrand to HTTPBase rather than HTTPBytes?
```

Comment:
We should probably resolve this question before merging, as this "brand" will be saved in the database.